Sources/Screens/RepositoryDetail/RepositoryDetailView.swift

```diff
@@ -1,11 +1,124 @@
+                            UIApplication.shared.open(url)
+                        }
+                    } label: {
+                        HStack {
+                            Text("\(viewModel.repository.name)")
+                                .textFontModifier(size: 20, weight: .bold)
+                                .foregroundColor(.black)
+                            Spacer()
+                            Image(systemName: "greaterthan")
+                        }
+                        .contentShape(Rectangle())
+                    }
+                }
+                Section("COUNTS") {
+                    HStack(alignment: .firstTextBaseline) {
+                        Text("⭐️")
+                            .countsEmojiModifier()
+                        Text("stars".capitalized)
+                        Spacer()
+                        Text("\(viewModel.repository.stargazersCount)")
+                    }
+                    HStack {
+                        Text("⫚")
+                            .countsEmojiModifier()
+                        Text("Forks")
+                        Spacer()
+                        Text("\(viewModel.repository.forks)")
+                    }
+                    HStack {
+                        Text("👀")
+                            .countsEmojiModifier()
+                        Text("Watchers")
+                        Spacer()
+                        Text("\(viewModel.repository.watchers)")
+                    }
+                }
+                Section("DESCRIPTION") {
+                    Text("Language: \(viewModel.repository.language ?? "n/a")")
+                        .textFontModifier(size: 20, weight: .semibold)
+                    Text(
+                        "Details: \(viewModel.repository.description ?? "n/a")"
+                    )
+                    .font(.title2)
+                }
+
+                Section("DATES") {
+                    HStack {
+                        Text(
+                            "Created at:"
+                        )
+                        .font(.headline)
+                        Spacer()
+                        Text("\(viewModel.repository.createdAt, style: .date)")
+                    }
+                    HStack {
+                        Text(
+                            "Updated at:"
+                        )
+                        .font(.headline)
+                        Spacer()
+                        Text(
+                            "\(viewModel.repository.updatedAt ?? Date.now, style: .date)"
+                        )
+                    }
+                }
+
+                Section("OWNER") {
+                    Button {
+                        guard let url = URL(
+                            string: viewModel.repository.owner
+                                .htmlUrl
+                        )
+                        else {
+                            return
+                        }
```

Comment:
On top button you use if let but here you use guard. Since it is identical case for both buttons, decide for one and be consistent.